README.md

```diff
@@ -92,8 +76,8 @@ To change the SNMP community from the default value and limit the netblocks that
 ```puppet
 class { 'snmp':
   agentaddress => [ 'udp:161', ],
-  ro_community => 'myPassword',
-  ro_network   => '192.168.0.0/16',
+  ro_community => ['myPassword'],
```

Comment:
I don't see the point in this as it breaks backward compatibility though in the next release you want to deprecate the option anyhow.

Reply:
Maybe this was a step too far.  I'll have another think.